3_javascript/javascript2/week1/scripts.js

```diff
@@ -0,0 +1,67 @@
+/// 1.2 Find and count the Danish letters ///
+
+function countDanishLetters(inputString) {
+  const danishLetters = {
+    æ: 0,
+    ø: 0,
+    å: 0,
+  };
+
+  for (const char of inputString) {
+    if (danishLetters[char]) {
+      danishLetters[char]++;
```

Comment:
it will never get here. The reason is that `danishLetters[char]` always returns the current value, which is 0 initially. And it is a `falsy` value in javascript (meaning it will always be false in the beginning.) Try to use the function `danishLetters.hasOwnProperty(char)`this will return true if the property exist.